src/integration-test/java/io/aiven/kafka/connect/http/IntegrationTest.java

```diff
@@ -285,7 +285,7 @@ final void testBatching() throws ExecutionException, InterruptedException {
                 sendFutures.add(sendMessageAsync(TEST_TOPIC, partition, key, value));
 
                 batchRecordCnt += 1;
-                currentBody.append(value).append("\n");
+                currentBody.append(value).append(BatchRecordSender.BATCH_RECORD_SEPARATOR);
```

Comment:
There are a couple of reasons to use a literal in places like this: 1) not expose privates; 2) actually see what we're testing and not relying on whatever (potentially erroneous) value we have in the constant.

Reply:
will bring it back